### PR TITLE
Spotify docs

### PR DESCRIPTION
**Pull request by vanFrZy**
Some simple documentation for spotify API and spotify control component for #37 .
Please let me know if we want certain information to be added/removed and I'll change it

### CODE REVIEW

File: src/Components/SpotifyControl/SpotifyControl.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ import useSpotifyAuth from "Store/selectors/useSpotifyAuthData"
 import bragiIcon from "Assets/images/bragi-icon.png"
 import SpotifyControlView from "./SpotifyControlView";
 
+/**
+ * Component for current song information showing play/pause button, cover image, name of song and artists
+ * @returns {JSX.Element} SpotifyControlView with current song information
+ * @constructor
```
Comment (Atema) [inline src/Components/SpotifyControl/SpotifyControl.js:10]: `@constructor` is supposed to be used for functions that are classes. Our components are just normal functions, so it should be removed.

File: src/Components/SpotifyControl/SpotifyControlView.js [hunk 2]
```diff
@@ -2,7 +2,16 @@ import styles from "./SpotifyControl.module.css"
...
-
+/**
+ * Simple song information view showing play/pause button, cover image, name of song and artists
+ * @param {function} onModifyPlayback Modifies current playback state
+ * @param {string} name current song name
+ * @param {string} imageSrc cover image for the current song
+ * @param {boolean} is_playing playing status of current song
+ * @param {string} artists artists of the current song
+ * @returns {JSX.Element} View with current song information
+ * @constructor
```
Comment (Atema) [inline src/Components/SpotifyControl/SpotifyControlView.js:13]: Same here

**Review by Atema — CHANGES_REQUESTED** [review]
Looks good 👍 One tiny thing that should be changed, see in-file comments, and feel free to merge afterward :)

File: src/Components/SpotifyControl/SpotifyControl.js (hunk 1 above)
Comment (vanFrZy) [reply src/Components/SpotifyControl/SpotifyControl.js:10]: Alright thanks, it was auto generated and wasn't sure what to do with it

Comment (Atema) [reply src/Components/SpotifyControl/SpotifyControl.js:10]: Ah, probably because the name starts with a capital, like classess usually

**Pull request merged**